uaclient/entitlements/base.py

```diff
@@ -194,6 +200,30 @@ def is_access_expired(self):
             return False
         return True
 
+    def process_contract_deltas(
+            self, orig_access: 'Dict[str, Any]',
+            deltas: 'Dict[str, Any]') -> None:
+        """Process any contract access deltas for this entitlement.
+
+        :param orig_access: Dictionary containing the original
+            resourceEntitlement access details.
+        :param deltas: Dictionary which contains only the changed access keys
+        and values.
+        """
+        if not deltas:
+            return
+        entitled_delta = deltas.get('entitlement', {}).get('entitled')
+        if orig_access and entitled_delta in (util.DROPPED_DICT_KEY, False):
+            if self.can_disable(silent=True):
```

Comment:
Should we do anything if we _can't_ disable? It might be helpful to inform the user that their status has changed? I'm not sure.

Reply:
This check serves two purposes:
1. don't try to disable something upon transition to unentitled if it isn't currently enabled
2. don't disable fips* entitlements which don't have a disable option.

I'll add an else here that also checks operational_status == ACTIVE to print an info message saying why it can't be disabled.